<ui version="4.0" >
 <class>ExportDlg</class>
 <widget class="QDialog" name="ExportDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Export</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>File name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_pFileNameE" />
     </item>
     <item>
      <widget class="QToolButton" name="m_pChooseFileB" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Format</string>
     </property>
     <layout class="QGridLayout" >
      <property name="topMargin" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="m_pXmlRB" >
        <property name="text" >
         <string>XML</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="m_pTextRB" >
        <property name="text" >
         <string>Text</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QRadioButton" name="m_pM3uRB" >
        <property name="text" >
         <string>M3U</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Remove root:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLineEdit" name="m_pM3uRootE" >
        <property name="toolTip" >
         <string>When creating an M3U file, this text gets removed from the beginning of the file names.
Meant to be used for creating M3U files containing relative paths.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Locale:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4" >
       <widget class="QComboBox" name="m_pLocaleCbB" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Files to save</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="topMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="m_pVisibleRB" >
        <property name="text" >
         <string>Visible files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pSelectedRB" >
        <property name="text" >
         <string>Selected files</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_pSortByShortNamesCkB" >
     <property name="text" >
      <string>Sort by short names</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pCloseB" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pExportB" >
       <property name="text" >
        <string>Export</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
